### ISSUE TITLE
Reader: Detail VC crashing when rotating 180 degrees

### ISSUE THREAD

**Issue by frosty** [wordpress-mobile/WordPress-iOS#6269]
I've found a reproducible crash with `WPRichContentView` in the splitview branch for the Reader (`feature/ipad-splitview-reader`). 

Reproduction steps:

* Open a post in the reader
* With the splitview expanded (so, iPad in regular width or iPhone Plus in landscape), flip the device 180 degrees (for example, landscape left to landscape right) in one motion.
* Crash.

Here's what I'm seeing:

```
2016-12-05 17:53:18.893 WordPress[66806:1521979] *** Assertion failure in -[NSLayoutConstraint _setSymbolicConstant:constant:], /BuildRoot/Library/Caches/com.apple.xbs/Sources/Foundation_Sim/Foundation-1349.1/Layout.subproj/NSLayoutConstraint.m:711
2016-12-05 17:53:18.921 WordPress[66806:1521979] *** Terminating app due to uncaught exception 'NSInternalInconsistencyException', reason: 'NSLayoutConstraint constant is not finite!  That's illegal.  constant:inf firstAnchor:<NSLayoutYAxisAnchor:0x61000046ba80 "UIStackView:0x7ffbb1e735e0.top"> secondAnchor:<NSLayoutYAxisAnchor:0x61000046a680 "WordPress.WPRichContentView:0x7ffbb62a2000'￼“Hopelessness is necessa...'.top">'
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000116f9734b __exceptionPreprocess + 171
	1   libobjc.A.dylib                     0x00000001169f821e objc_exception_throw + 48
	2   CoreFoundation                      0x0000000116f9b442 +[NSException raise:format:arguments:] + 98
	3   Foundation                          0x00000001111a2e4d -[NSAssertionHandler handleFailureInMethod:object:file:lineNumber:description:] + 195
	4   Foundation                          0x0000000111122353 -[NSLayoutConstraint _setSymbolicConstant:constant:] + 269
	5   Foundation                          0x0000000111122217 -[NSLayoutConstraint setConstant:] + 68
	6   WordPress                           0x000000010e594492 _TFC9WordPress26ReaderDetailViewControllerP33_6894C07EBB00C79B78761FE3A823FACE21updateTextViewMarginsfT_T_ + 1010
	7   WordPress                           0x000000010e5911d9 _TFFC9WordPress26ReaderDetailViewController24viewWillTransitionToSizeFTVSC6CGSize25withTransitionCoordinatorPSo37UIViewControllerTransitionCoordinator__T_U_FPSo44UIViewControllerTransitionCoordinatorContext_T_ + 41
	8   WordPress                           0x000000010e591217 _TTRXFo_oPSo44UIViewControllerTransitionCoordinatorContext__dT__XFo_iPS___iT__ + 23
	9   WordPress                           0x000000010e5a28a1 _TPA__TTRXFo_oPSo44UIViewControllerTransitionCoordinatorContext__dT__XFo_iPS___iT__ + 81
	10  WordPress                           0x000000010e591160 _TTRXFo_iPSo44UIViewControllerTransitionCoordinatorContext__iT__XFo_oPS___dT__ + 32
	11  WordPress                           0x000000010e5911a8 _TTRXFo_oPSo44UIViewControllerTransitionCoordinatorContext__dT__XFdCb_dPS___dT__ + 56
	12  UIKit                               0x0000000112b25539 -[_UIViewControllerTransitionCoordinator _applyBlocks:releaseBlocks:] + 311
	13  UIKit                               0x0000000112b212f1 -[_UIViewControllerTransitionContext _runAlongsideCompletions] + 151
	14  UIKit                               0x0000000112b20fcd -[_UIViewControllerTransitionContext completeTransition:] + 118
	15  UIKit                               0x000000011267a033 __58-[_UIWindowRotationAnimationController animateTransition:]_block_invoke_2.206 + 26
	16  UIKit                               0x000000011209a7e4 +[UIView(Internal) _performBlockDelayingTriggeringResponderEvents:] + 188
	17  UIKit                               0x0000000112679fe4 __58-[_UIWindowRotationAnimationController animateTransition:]_block_invoke.205 + 124
	18  UIKit                               0x00000001120956aa -[UIViewAnimationBlockDelegate _didEndBlockAnimation:finished:context:] + 528
	19  UIKit                               0x0000000112068475 -[UIViewAnimationState sendDelegateAnimationDidStop:finished:] + 222
	20  UIKit                               0x00000001120689ca -[UIViewAnimationState animationDidStop:finished:] + 136
	21  QuartzCore                          0x0000000111b71990 _ZN2CA5Layer23run_animation_callbacksEPv + 316
	22  libdispatch.dylib                   0x0000000117e7c0cd _dispatch_client_callout + 8
	23  libdispatch.dylib                   0x0000000117e5c8d6 _dispatch_main_queue_callback_4CF + 406
	24  CoreFoundation                      0x0000000116f5b4f9 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
	25  CoreFoundation                      0x0000000116f20f8d __CFRunLoopRun + 2205
	26  CoreFoundation                      0x0000000116f20494 CFRunLoopRunSpecific + 420
	27  GraphicsServices                    0x0000000119a0da6f GSEventRunModal + 161
	28  UIKit                               0x0000000111fdb964 UIApplicationMain + 159
	29  WordPress                           0x000000010e29e8be main + 62
	30  libdyld.dylib                       0x0000000117ec868d start + 1
)
```

cc @aerych

**Comment by aerych** [comment]
> NSLayoutConstraint constant is not finite!  That's illegal.

🙀  That's a new one.

**Comment by aerych** [comment]
Dug into this.  The crash is actually with the top constraint for the attribution view.  More of a detail vc issue than an issue with the content view, but that's just me being pendantic. :P 
Patch otw.

**Comment by aerych** [comment]
Closed in #6274
